Sources/EdgeBridgeConstants.swift

```diff
@@ -20,8 +20,11 @@ enum EdgeBridgeConstants {
 
     enum EventDataKeys {
         enum ContextDataKeys {
-            static let CONTEXT_DATA_OUTPUT_WITH_MERGE = "withMerge"
-            static let CONTEXT_DATA_MERGE_IS_CASE_SENSITIVE = "isMergeCaseSensitive"
+            static let CAPTURE_STATE = "captureState"
+            static let CAPTURE_STATE_START = "start"
+            static let CAPTURE_STATE_STOP = "stop"
```

Comment:
@timkimadobe we could actually skip everything and just keep one flag again. Let's say Capture based on its value true/false we can start/stop. 

So the event with type edgebridge and source captureContent
event data would have 
[
"capture": true, 
"merge" : true
 "caseSensitiveMerge": true
 ]
 
 PS: we could simplify the keys as mentioned above.

Reply:
Yes it's better to send on stop.